packages/terra-form-validation/docs/NumericValidation.md

```diff
@@ -0,0 +1,177 @@
+
+Second thing - our validation function, `validateNumber`, will be given a *string*, despite the input being of `type: 'number'`. This does not impact simple operators but is useful to know when building custom validation functions.
+
+Let's start by giving a minimum and maximum to our value. For simplicity, set a minimum to 10 and a maximum to 100.
+
+```diff
+const validateNumber = (value) => {
+  if (!value) { return undefined; }
++ if (value < 10) {
++   return 'Value should not be less than 10.';
++ }
++ if (value > 100) {
++   return 'Value should not be over 100.';
++ }
+
+  return undefined;
+};
+```
+
+Now, let's check for precision. For this example, lets validate to the thousandth place (three decimal places). Since our value is a string, we'll use string functions to validate.
+
+Use `.` as a delimiter and check for a length less than or equal to 3.
+
+```diff
+const validateNumber = (value) => {
+  ...
++ const valueSplit = value.split('.');
++ if (valueSplit.length === 2 && valueSplit[1].length > 3) {
++   return 'Value has more than 3 decimal points';
++ }
+
+  return undefined;
+};
+```
+
+This validation has a quirk. If the input contains extra `0`s, like `23.0230000`, validation will fail.
+
+Lets try a different way of validating. Convert the value a `Number` instead. This way we can use the `toFixed` function to automatically set precision, and use this as a comparison.
+
+```diff
+const validateNumber = (value) => {
+  ...
+- const valueSplit = value.split('.');
+- if (valueSplit.length === 2 && valueSplit[1].length > 3) {
++ const valueNum = Number(value);
++ if (valueNum.toFixed(3) != valueNum) {
+    return 'Value has more than 3 decimal points';
+  }
+
+  return undefined;
+};
+```
+
+This validation has the opposite quirk. If the input contains extra `0`s, like `23.0230000`, validation will succeed.
+
+With this we have a function that we can add to validate a number based by having a minimum, maximum, and a certain level of precision required. In the same vein as the 'Max Length Input' example, we can use the `FormValidationUtil` to help control and reduce some of workflow.
+
+```diff
+const validateNumber = (value) => {
+-  if (value < 10) {
++  if (!FormValidationUtil.isOverMinValue(value, 10)) {
+    return 'Value should not be less than 10.';
+  }
+-  if (value > 100) {
++  if (!FormValidationUtil.isUnderMaxValue(value, 100)) {
+    return 'Value should not be over 100.';
+  }
+- const valueNum = Number(value);
+- if (valueNum.toFixed(3) != valueNum) {
++ if (!FormValidationUtil.isPrecise(value, 3)) {
+    return 'Value has more than 3 decimal points';
+  }
+
+  return undefined;
+};
+```
+
+Note: `FormValidationUtil.precisionCheck` uses the first method check precision.
```

Comment:
Hmm, Im not sure what this means.

Reply:
Cleared it up here https://github.com/cerner/terra-framework/pull/635/commits/415c46ab9a0b74893a686c38430b1d8decd708ad#diff-efb0b388a04429df36f1ce4eb79ef872R108